doc/ref_arch/kubernetes/chapters/chapter03.md

```diff
@@ -63,7 +63,7 @@ Because `req.inf.ntw.01` requires the architecture to support CNI and `req.inf.n
 
 A CNI metaplugin/CNI multiplexer is capable to attach several interfaces, using different other CNI plugins, to a pod. The different network charastheristics of the interfaces, which require different networking technologies require different CNI plugins.
 
-Inter node communication required by `req.inf.ntw.02` must be served by a CNI plugin with basic networking capabilities, like Calico or Flannel.
+Inter node communication required by `req.inf.ntw.02` must be served by a CNI plugin which complies with the default K8s networking assumptions.
```

Comment:
is it "assumptions" or (default k8s networking) "model?"

Reply:
It is more like assumptions. like what Kubernetes assumes to happen on the given network.